### ISSUE TITLE
Add stat that allows attackers to take more damage from defenders when attacking

### ISSUE THREAD

**Issue by IceRaptor** [BattletechModders/CBTBehaviorsEnhanced#77]
LA would like a stat that allows defenders to do damage when attacked in melee. Think 'spiked armor' or equivalent.

**Comment by IceRaptor** [comment]
Alternatively:

we gotta put him in one of these
ohhh @FrostRaptor i have a idea for charge balance
have defender/target tons affect attacker self damage taken
that could tie into my ideas with the spikes too and scale that
Interactive Rubber Dolphin11/16/2020
that's how charge is calculated
????
Damage to attacker is calculated by target tons
@Alekto
Alekto11/16/2020
oh?
Interactive Rubber Dolphin11/16/2020
"Charge": {
                "AttackerDamagePerTargetTon": 0.5,
Alekto11/16/2020
so basically for my spikes idea
i only need a way to scale that on the defender
